agent/dockerclient/dockerapi/docker_client.go

```diff
@@ -1266,11 +1266,11 @@ func (dg *dockerGoClient) listPlugins(ctx context.Context) ListPluginsResponse {
 
 // APIVersion returns the client api version
 func (dg *dockerGoClient) APIVersion() (dockerclient.DockerVersion, error) {
-	client, err := dg.dockerClient()
+	client, err := dg.sdkDockerClient()
```

Comment:
Since we have two different clients, how do we know if `APIVersion()` is returning the sdk client's version?